packages/captp/src/captp.js

```diff
@@ -26,8 +32,16 @@ export { E };
  * @property {typeof defaultRemotable} Remotable
```

Comment:
Now that captp has its own types.js, should substantial types like this move there?

Reply:
I can't move this one until `@agoric/marshal` no longer uses ambient types.  But I did improve the typing a little bit.

Reply:
Resolving